src/windows/multiplayer.c

```diff
@@ -264,7 +264,7 @@ static void window_multiplayer_groups_show_group_dropdown(rct_window *w, rct_wid
 
 	for (i = 0; i < network_get_num_groups(); i++) {
 		gDropdownItemsFormat[i] = 2777;
-		gDropdownItemsArgs[i] = (sint64)network_get_group_name(i);
+		gDropdownItemsArgs[i] = (int)network_get_group_name(i);
```

Comment:
We might forget to change it though, what happens if you cast it to ```size_t``` first then ```sint64```?